addons/pkg/types/clusterbootstrap.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package types defines type constants.
+package types
+
+const (
+	// HTTPProxyConfigAnnotation is the cluster HTTP proxy annotation key
+	HTTPProxyConfigAnnotation = "tkg.tanzu.vmware.com/tkg_http_proxy"
```

Comment:
The reason for keeping both `HTTPProxyConfigAnnotation` and `DefaultIPFamilyClusterClassVarName` is that even though they keep the same value, they have different semantic meanings.